src/ray/raylet/scheduling/policy/bundle_scheduling_policy.cc

```diff
@@ -424,7 +424,7 @@ SchedulingResult BundleStrictPackSchedulingPolicy::Schedule(
           const auto &entry) {
         const auto &node_resources = entry.second->GetLocalView();
         auto allocatable =
-            (node_resources.IsAvailable(
+            (node_resources.IsFeasible(
```

Comment:
Isn't it supposed to be feasible + available? (if there's no available resource, we cannot allocate it? )

Reply:
hmm why is it resolved?